python/whylogs/api/writer/whylabs.py

```diff
@@ -66,20 +67,32 @@ def __init__(self, org_id: Optional[str] = None, api_key: Optional[str] = None,
         self._api_key = api_key or os.environ.get("WHYLABS_API_KEY")
         self._dataset_id = dataset_id or os.environ.get("WHYLABS_DEFAULT_DATASET_ID")
         self.whylabs_api_endpoint = os.environ.get("WHYLABS_API_ENDPOINT") or "https://api.whylabsapp.com"
+        self._reference = False
+        self._reference_profile_alias = os.environ.get("WHYLABS_REFERENCE_PROFILE_ALIAS")
 
     def check_interval(self, interval_seconds: int):
         if interval_seconds < FIVE_MINUTES_IN_SECONDS:
             raise BadConfigError("Bad WhyLabsWriter config: interval must be greater or equal to five minutes")
 
     def option(
-        self, org_id: Optional[str] = None, dataset_id: Optional[str] = None, api_key: Optional[str] = None
-    ) -> None:
+        self,
+        org_id: Optional[str] = None,
+        dataset_id: Optional[str] = None,
+        api_key: Optional[str] = None,
+        reference: Optional[bool] = None,
+        reference_alias: Optional[str] = None,
```

Comment:
How about dropping the bool parameter and just inferring this from the name given?
```suggestion
        reference_profile_name: Optional[str] = None,
```

Reply:
That's a good idea!

Just one thing: the WhyLabs API allows you to create a reference profile without a name. In this case, the name is the timestamp of when it is created.

Inferring from the name would prevent the user of logging a reference profile without a name, even though the API allows it. Do we want that?

Reply:
Use `""` for an unnamed reference profile, and `None` for "not a reference profile?"

Reply:
ah, sure. I did try this before but didn't realize it was just a tweak to make it work.
changed this in the last commmit.